scripts/build_climate_projections.py

```diff
@@ -0,0 +1,210 @@
+# -*- coding: utf-8 -*-
+# SPDX-FileCopyrightText:  PyPSA-Earth and PyPSA-Eur Authors
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# -*- coding: utf-8 -*-
+"""
+Creates a dataset corresponding to the projected climate in a requested year.
```

Comment:
Good that you already thought of the docstring :D
As a little remark, it could be nice to add few lines of description of the methodology, taht falicitates the review as well

Reply:
Thanks :) Agree, it's definitely worth to explain the method (it's very simple, in fact) and the reasoning behind.

Reply:
Done :)